backend/integration_tests/test/test_app.js

```diff
@@ -5,7 +5,7 @@ const { expect } = chai;
 
 chai.use(chaiHttp);
 
-const url = process.env.API_URL || 'http://app:8000/';
+const url = process.env.API_URL || 'http://localhost:8000/';
```

Comment:
Did you forget to add `/api` here?

Reply:
> 
> 
> Did you forget to add `/api` here?
This one is overriden in docker-compose.yml, so tests will pass. But i guess I'll change it here as well, to avoid confusion.